inst/pages/19_visualization_techniques.qmd

```diff
@@ -276,41 +276,39 @@ at the Phylum level are available in the aforementioned chapter.
 ```{r}
 #| label = "plotAbundance1"
 # agglomerate tse by Order
-tse_order <- mergeFeaturesByRank(tse,
-                                rank = "Order",
-                                onRankOnly = TRUE)
+tse_phylum <- mergeFeaturesByRank(tse,
+                                  rank = "Phylum",
+                                  onRankOnly = TRUE)
+
```

Comment:
How about adding this Phylum-level thing into an altExp instead of creating a new TreeSE object?